json_logic/__init__.py

```diff
@@ -122,11 +130,18 @@ def get_var(data, var_name, not_found=None):
 
 
 def get_date(date_str, *args):
-    try:
-        return date.fromisoformat(date_str)
-    except ValueError:
-        date_with_time = datetime.fromisoformat(date_str)
-        return date_with_time.date()
+    warnings.warn("The date operator is deprecated", DeprecationWarning)
+    return date_str
```

Comment:
deprecating is okay, but the behaviour has also changed to return a string now instead of date/datetime object. is that right?

Reply:
Yes, I wasn't sure what the best way to go about it is, since now the `{"date": ...}` operator and the `{"today": []}` shouldn't be used anymore. But I didn't want all the existing logic rules not to work any more :thinking: